webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Immutable from 'seamless-immutable';
+import { TypeAheadSelect as PfTypeAheadSelect } from 'patternfly-react';
+import { updateSelected } from './TypeAheadSelectActions';
+
+const TypeAheadSelect = ({
+  id,
+  options,
+  defaultSelected,
+  selected,
+  allowNew,
+  multiple,
+}) => {
+  const dispatch = useDispatch();
+
+  const mutableOptions = Immutable.isImmutable(options)
+    ? options.asMutable()
+    : options;
+
+  return (
+    <PfTypeAheadSelect
```

Comment:
can you add here the ability to pass more props of `PfTypeAheadSelect` for other usages?